tools/taskcluster/decision_task.py

```diff
@@ -19,8 +19,13 @@ def master(builder):
     return (taskcluster.slugId(), builder.craft_master_task()),
 
 
-def release(builder, tag):
-    return (taskcluster.slugId(), builder.craft_release_build_task(tag)),
+def release(builder, tag, is_staging):
+    build_task_id = taskcluster.slugId()
+    return (
+        (build_task_id, builder.craft_release_build_task(tag)),
+        (taskcluster.slugId(), builder.craft_sign_for_github_task(build_task_id, is_staging)),
```

Comment:
Does `taskcluster.slugId()` generate new ids? Or should be you using `build_task_id` here and below?

Reply:
`taskcluster.slugId()` generates new ids: the first item of each of these tuples is the task id associated with each task.

You can imagine it like:
```
return (
    ('A', build_task()),
    ('B', sign_task('A')),
    ('C', push_task('A')),
    ('D', email_task('B', 'C')),
)
```